flink-runtime-web/web-dashboard/src/app/pages/task-manager/metrics/task-manager-metrics.component.html

```diff
@@ -58,7 +59,7 @@
         </td>
       </tr>
       <tr>
-        <td class="table-header">Task Off-Heap</td>
+        <td class="table-header">Netty Shuffle BuffersTask Off-Heap</td>
```

Comment:
`Netty Shuffle Buffers` shouldn't be here.

Reply:
done, sorry for the mistaken